### DISCUSSION TITLE
syntax error?

### DISCUSSION THREAD

**Discussion by theposs101** [magicbug/Cloudlog#1941] [Q&A]
![wtf](https://user-images.githubusercontent.com/9128916/213527033-e9baa5fa-5cde-4f19-a3cd-955eedd41c4c.png)

i have this really weird error, if anyone knows the solution or has had the same issue, id appreciate the help!

**Comment by phl0** [comment]
Which version/branch is it?

**Reply by phl0** [reply]
Never mind. Found it. It is a simple syntax error. Fix proposed with https://github.com/magicbug/Cloudlog/pull/1942.

**Reply by phl0** [reply]
Just out of curiosity: Which callsign did you enter to provoke that error?

**Comment by theposs101** [comment]
I actually got the error while trying to install.

It’s an Ubuntu 18.04 box with Apache2 and php7.2 and the needed modules.
